src/mpyl/cli/build.py

```diff
@@ -116,8 +125,24 @@ def build(ctx, config, properties, verbose):
     help="Build all projects, regardless of changes on branch",
 )
 @click.option("--tag", "-t", help="Tag to build", type=click.STRING, required=False)
+@click.option(
+    "--stage",
+    default=None,
+    type=click.Choice([stage.value for stage in Stage]),
```

Comment:
This should come from the dynamic list of stages once https://github.com/Vandebron/mpyl/pull/255 is merged.